examples/trials/mnist-cascading-search-space/mnist.py

```diff
@@ -131,21 +131,29 @@ def main(params):
 
         nni.report_final_result(test_acc)
 
-def generate_defualt_params():
-    params = {'data_dir': '/tmp/tensorflow/mnist/input_data',
-              'batch_num': 1000,
-              'batch_size': 200}
-    return params
-
+def get_params():
+    ''' Get parameters from command line '''
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--data_dir", type=str, default='/tmp/tensorflow/mnist/input_data', help="data directory")
+    parser.add_argument("--batch_num", type=int, default=1000)
+    parser.add_argument("--batch_size", type=int, default=200)
+    args, _ = parser.parse_known_args()
+    return args
 
 def parse_init_json(data):
     params = {}
     for key in data:
         value = data[key]
-        if value == 'Empty':
+        layer_name = value["_name"]
+        if layer_name == 'Empty':
+            # Empty Layer
             params[key] = ['Empty']
+        elif layer_name == 'Conv':
+            # Conv layer
+            params[key] = [layer_name, value['kernel_size'], value['kernel_size']]
```

Comment:
why input two "value['kernel_size']"?

Reply:
One is the kernel width and one is the kernel height.